src/clients/repositories.rb

```diff
@@ -641,10 +642,20 @@ def buildList
             @displayed_service == Ops.get_string(srv_state, "alias", "")
         )
         ret = Builtins.add(ret, t)
-      end 
+      end
 
+      # there is some service, so allow to filter repos without service (bnc#944504)
+      if ret.size > 2
+        t = Item(
+          Id(:without_service),
+          _("Without Service"),
```

Comment:
It's the other way round, a service contains repositories. The filter should be named "Repositories not in a service",  "Repositories not belonging to a service" or something like this. A bit longer but I'm not sure if there is a good shorter alternative...

@mvidner any suggestion here?